src/main/java/org/plumelib/util/FilesPlume.java

```diff
@@ -64,7 +65,7 @@ private FilesPlume() {
    * @return an InputStream for file
    * @throws IOException if there is trouble reading the file
    */
-  public static InputStream newFileInputStream(Path path) throws IOException {
+  public static @Owning InputStream newFileInputStream(Path path) throws IOException {
```

Comment:
Did you find that you needed these explicit `@Owning` annotations on return types?  The checker should treat returns as `@Owning` by default, so I am surprised you needed to write these.

Reply:
You are right that I don't need them.  I must have been confused at some point.  Thanks for simplifying the pull request!